react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+
+export default function ToggleSwitch() {
+  const switchClasses = {
+    text: 'OFF',
+    circle: 'circle',
+    boxBackground: 'grey',
+    off: ''
+  };
+
+  const [toggle, setToggle] = useState(false);
+  function handleClick() {
+    setToggle(!toggle);
+  }
+
+  function checkToggle() {
+    if (toggle) {
+      switchClasses.text = 'ON';
+      switchClasses.toggle = 'on';
+      switchClasses.boxBackground = 'on-background';
+    } else {
+      switchClasses.text = 'OFF';
+      switchClasses.toggle = '';
+      switchClasses.boxBackground = 'off-background';
+    }
+  }
+  checkToggle();
```

Comment:
You could just inline the logic inside of `checkToggle` instead of declaring a function.